python/strings_udf/strings_udf/tests/test_string_udfs.py

```diff
@@ -68,7 +69,8 @@ def run_udf_test(data, func, dtype):
     str_views = column_to_string_view_array(cudf_column)
 
     kernel = get_kernel(func, dtype, len(data))
-    kernel.forall(len(data))(str_views, output)
+    with acquire_spill_lock():
+        kernel.forall(len(data))(str_views, output)
```

Comment:
I think we should avoid setting this precedent; tests really shouldn't have to worry about spilling.

Reply:
Discussed offline and removed this lock to keep things consistent where we are explicitly not testing for spilling ability.